src/components/notebook-utils.js

```diff
@@ -91,7 +91,7 @@ export const toolExtensionDisplay = {
 }
 
 export const getPatternFromTool = toolLabel => Utils.switchCase(toolLabel,
-  [tools.RStudio.label, () => '.*\\.R(md)'],
+  [tools.RStudio.label, () => '.+(.R|.Rmd)$'],
```

Comment:
`.`s inside the parentheses need escaping or else the expression would match things like `abR`.

```suggestion
  [tools.RStudio.label, () => '.+(\.R|\.Rmd)$'],
```

Reply:
Oh, interesting. I wasn't sure if they still needed escaping when inside `()`s.